### PR TITLE
Edit time label

### PR DESCRIPTION
**Pull request by kdh0324**

### CODE REVIEW

**Comment by kdh0324** [comment]
Replace slider and time label position

**Comment by kdh0324** [comment]
Use absolute position of QLabel(video streamer) to express the position of mouse cursor

**Comment by SangbumChoi** [comment]
@kdh0324 two big issues two solve
1. (important) timebar (which indicates the showing time does not directly connected to video progressbar, former same issue compare to video progress bar and playing widget)
2. (less important) first we stop the video, and move progressbar to end of the video, timer shows 00:00:03

**Comment by kdh0324** [comment]
Add tick in 5s

**Comment by kdh0324** [comment]
Edit video saving function to use list.
I edited video streaming by resizing during streaming the video inevitably.

**Comment by kdh0324** [comment]
Add total time label

**Comment by kdh0324** [comment]
Solve error #38

**Comment by SangbumChoi** [comment]
@kdh0324 saving looks fine, however although I download into avi file it automatically saves with mp4
(minor problem) making QProgressBar I will comment it please check


One for @SangbumChoi I've pre-processed QImage and resizing part on run_demo and I have to delete it. (Test Works when I delete this code)

File: VideoSave/VideoSave.py [hunk 1]
```diff
@@ -25,28 +29,27 @@ def initUI(self):
...
-        self.progress.setMaximum(cap.get(cv2.CAP_PROP_FRAME_COUNT))
+        self.progress.setMaximum(len(self.frameList))
+
+        while count < len(self.frameList):
+            (ret, frame) = self.frameList[count]
 
-        while cap.isOpened():
-            ret, frame = cap.read()
             if ret:
                 count += 1
```
Comment (SangbumChoi) [inline VideoSave/VideoSave.py]: In this algorithm looping with 100 times will give us the progress bar with done.

I think you should initialize the end value of progressbar into total frame.

**Pull request merged**